tests/risks/test_data_transformations.py

```diff
@@ -31,11 +31,11 @@ def test__rebin_exposure_data(rebin_categories, rebinned_values):
             "value": [0.5] * 4 + [0.2] * 4 + [0.3] * 4,
         }
     )
-    rebinned_df = _rebin_exposure_data(df, rebin_categories)
+    rebinned_df = DichotomousDistribution._rebin_exposure_data(df, rebin_categories)
```

Comment:
This confused me b/c there actually is still a data_transformations.rebin_exposure_data utility function. Can they be combined or are they just too different?

Reply:
They are basically identical, but the methods in data_transformations.py only still exist because they are called by get_relative_risk_data. Once I do the refactor of RiskEffect they will all go away.